src/unix/bsd/netbsdlike/netbsd/mod.rs

```diff
@@ -658,6 +673,11 @@ extern {
     pub fn newlocale(mask: ::c_int,
                      locale: *const ::c_char,
                      base: ::locale_t) -> ::locale_t;
+
+    // This should work, but it causes the netbsd CI build to fail with an
+    // intra-libc.a undefined reference to `wait6`.
+    //pub fn waitid(idtype: idtype_t, id: id_t, infop: *mut ::siginfo_t,
+    //              options: ::c_int) -> ::c_int;
```

Comment:
This may just need a `#[link_name]` of some form, but otherwise it's ok to just delete this for now.